doc/source/whatsnew/v1.5.2.rst

```diff
@@ -27,7 +27,7 @@ Fixed regressions
 Bug fixes
 ~~~~~~~~~
 - Bug in the Copy-on-Write implementation losing track of views in certain chained indexing cases (:issue:`48996`)
--
+- Bug with python 3.11 and creating a subclass that is ``Generic`` (:issue:`49649`)
```

Comment:
Maybe make it clearer that it is `typing.Generic`:

`Fixed bug when instantiating a ``DataFrame`` subclass inheriting from ``typing.Generic`` that triggered a ``UserWarning`` on python 3.11 (:issue:`49649`)`

Reply:
Good suggestion.  Fixed in https://github.com/pandas-dev/pandas/pull/49736/commits/c62461db793d5cd7038b5b0cb43a931b6eb45038